spec/models/customer_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe Customer do
+  it { is_expected.to validate_presence_of(:name) }
+  it { is_expected.to validate_presence_of(:urn) }
+  it { is_expected.to validate_presence_of(:sector) }
```

Comment:
How about a `validate_uniqueness_of(:urn)` test here for completeness?

Reply:
adding for completeness, although I'll be honest I'm not a huge fan of these declarative specs. not sure what value they add as they are simply mirror the validations on the model.